src/frontend/src/lib/components/convert/ConvertNetwork.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+	import ConvertValue from '$lib/components/convert/ConvertValue.svelte';
+	import Logo from '$lib/components/ui/Logo.svelte';
+	import { i18n } from '$lib/stores/i18n.store.js';
+	import type { Token } from '$lib/types/token';
+	import { replacePlaceholders } from '$lib/utils/i18n.utils.js';
+
+	export let token: Token;
+</script>
+
+<ConvertValue>
+	<slot slot="label" name="label" />
+
+	<div class="flex" slot="main-value">
+		<Logo
```

Comment:
not for this PR, but I am pretty sure we are using the same network BW logo elsewhere too, on more occasions. Is it worth to create a component for it?

Reply:
Sounds good, the next time it's needed - let's create a common component for that

Reply:
Done with:

- PR #3604

Reply:
Nice, thanks!